Jint.Tests/Runtime/ModuleTests.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Jint.Native;
+using Jint.Runtime;
+using Xunit;
+
+namespace Jint.Tests.Runtime
+{
+    public class ModuleTests
+    {
+        private readonly Engine _engine;
+
+        public ModuleTests()
+        {
+            _engine = new Engine();
+        }
+
+        [Fact]
+        public void ShouldExportNamed()
+        {
+            _engine.DefineModule(@"export const value = 'exported value';", "my-module");
+            var ns = _engine.ImportModule("my-module");
```

Comment:
Does `ImportModule` actually import a module in the environment, or does it just return it to the caller, and doing an `Execute('return value')` would not work.